### PR TITLE
8381 - Added setting to disable nav switcher

### LINKED ISSUE #8381: ModuleNav: Disable Module Nav Switcher
**Is your feature request related to a problem? Please describe.**
There's no option to disable the module-nav-switcher so that the user can't interact with it. 

**Describe the solution you'd like**
Being able to see an icon and a text instead of a dropdown

**Describe alternatives you've considered**
Hiding the whole section, however, this may cause confusion.

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by yohannahbautista**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

Added setting to disable nav switcher

**Related github/jira issue (required)**:

Closes #8381 

**Steps necessary to review your pull request (required)**:

- Pull this branch, build and run the app
- Go to: http://localhost:4000/components/module-nav/example-index.html
- Toggle `Disable switcher` checkbox
- Switcher should be disabled/enabled on check/uncheck

**Included in this Pull Request**:
- [ ] A note to the change log.

### CODE REVIEW

**Review by tmcconechy — CHANGES_REQUESTED** [review]
@yohannahbautista when its disabled can we make the style

- no background
- 20px size (same as header)

Screenshot 2024-04-01 at 9 52 09 AM

**Review by ericangeles — APPROVED** [review]
Add this setting to NG as well. Thanks, @yohannahbautista 🍌

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@
 - `[Homepage]` Added row height style classes for widgets. ([#8211](https://github.com/infor-design/enterprise/issues/8211))
 - `[Lookup]` Added placeholder setting in toolbar. ([#8416](https://github.com/infor-design/enterprise/issues/8416))
 - `[Modal]` Added setting `draggable` to be able to drag modals. ([#7019](https://github.com/infor-design/enterprise/issues/7019))
+- `[Module Nav]` Added setting `disableSwitcher` to disable nav switcher. ([#8381](https://github.com/infor-design/enterprise/issues/8381))
```
Comment (ericangeles) [inline docs/CHANGELOG.md:12]: Lets put it on `4.95` log, @yohannahbautista

**Comment by ericangeles** [comment]
Let's not merge it yet @tmcconechy once this is okay with the QA.

**Review by tmcconechy — APPROVED** [review]

**Comment by jbrcna** [comment]
this is now working as expected
image

However, in firefox the text is not centered align.
image

**Review by tmcconechy — APPROVED** [review]
Made a minor tweak as per https://inforwiki.atlassian.net/browse/IDS-2135
Approved now from design

**Review by jbrcna — APPROVED** [review]

**Review by n-ace-ancog — APPROVED** [review]
Working as expected.
Animation

**Review by glenlieorillo — APPROVED** [review]
https://github.com/infor-design/enterprise/assets/102151352/78fdbbcd-33ea-42f9-a89f-4d8469e951e1

**Pull request merged**